src/table/use-column-widths.tsx

```diff
@@ -16,15 +16,18 @@ export interface ColumnWidthDefinition {
   width?: string | number;
 }
 
+const treatAsNumber = (value: number | string | undefined) =>
+  typeof value === 'number' || (value && parseInt(value, 10).toString() === value);
```

Comment:
Wouldn't it be better to check if `parseInt` returns `NaN` using the built-in `isNaN` function?

Reply:
that wouldn't work because `parseInt('10%')` returns `10` rather than `NaN`, and we explicitly want to treat that sort of string as a string, not a number

Reply:
Got it. In this case, I would put this function in utils and cover it with unit tests. WDYT?

Reply:
sounds a good plan!